Rexray won't start on Ubuntu Xenial

# Summary
Cannot start rexray on Ubuntu Xenial :
```shell
$ curl -sSL https://dl.bintray.com/emccode/rexray/install | sh
Selecting previously unselected package rexray.
(Reading database ... 75619 files and directories currently installed.)
Preparing to unpack rexray_0.10.0-1_amd64.deb ... 
Unpacking rexray (0.10.0-1) ... 
Setting up rexray (0.10.0-1) ... 

rexray has been installed to /usr/bin/rexray

REX-Ray
-------
Binary: /usr/bin/rexray
Flavor: client+agent+controller
SemVer: 0.10.0
OsArch: Linux-x86_64
Commit: c23d059e0a3e1dd43a517d5bf7d6f5032b13ded5
Formed: Mon, 11 Sep 2017 02:16:48 CEST

$ rexray volume ls -l
panic: runtime error: index out of range

goroutine 1 [running]:
github.com/codedellemc/rexray/util.FindFlagVal(0x25e0aed, 0x2, 0xc420010180, 0x4, 0x4, 0x0, 0x24939a0, 0x1, 0x2, 0x90)
        /go/src/github.com/codedellemc/rexray/util/util.go:573 +0x23d
main.main()
        /go/src/github.com/codedellemc/rexray/rexray.go:64 +0x186
$ lsb_release -a
No LSB modules are available.
Distributor ID: Ubuntu
Description:    Ubuntu 16.04.3 LTS
Release:        16.04
Codename:       xenial
```



## Version
Please paste the output of `rexray version`. For example:

```shell
# rexray version
REX-Ray
-------
Binary: /usr/bin/rexray
Flavor: client+agent+controller
SemVer: 0.10.0
OsArch: Linux-x86_64
Commit: c23d059e0a3e1dd43a517d5bf7d6f5032b13ded5
Formed: Mon, 11 Sep 2017 02:16:48 CEST

```

## Logs
### Service Log
empty (just the logo)

--- a/util/util.go
+++ b/util/util.go
@@ -558,7 +558,7 @@ func parseSafeHost(ctx apitypes.Context, h string) string {
 func FindFlagVal(name string, args ...string) (string, []int) {
 	var format string
 	if strings.HasPrefix(name, "--") {
-		format = `(?i)^%s(?:=(.+))?$`
+		format = `(?i)^%s(?:=(.*))?$`
 	} else {
 		format = `(?i)^%s$`
 	}
@@ -569,12 +569,16 @@ func FindFlagVal(name string, args ...string) (string, []int) {
 		case 0:
 			continue
 		case 1:
-			if i+1 <= len(args) {
+			if i+1 < len(args) {
 				return args[i+1], []int{i, i + 1}
 			}
+			return "", []int{i}
 		case 2:
 			if m[1] == "" {
-				return args[i+1], []int{i, i + 1}
+				if i+1 < len(args) {
+					return args[i+1], []int{i, i + 1}
+				}
+				return "", []int{i}
 			}
 			return m[1], []int{i}
 		}

--- a/util/util_test.go
+++ b/util/util_test.go
@@ -175,4 +175,37 @@ func TestFindFlagArgs(t *testing.T) {
 			t.Fatalf("invalid indices: %v", indices)
 		}
 	}
+
+	{
+		val, indices := util.FindFlagVal(
+			"-l", "rexray", "-l")
+		if val != "" {
+			t.Fatalf("val != '': %s", val)
+		}
+		if len(indices) != 1 && indices[0] != 1 {
+			t.Fatalf("invalid indices: %v", indices)
+		}
+	}
+
+	{
+		val, indices := util.FindFlagVal(
+			"--logLevel", "rexray", "--logLevel")
+		if val != "" {
+			t.Fatalf("val != '': %s", val)
+		}
+		if len(indices) != 1 && indices[0] != 1 {
+			t.Fatalf("invalid indices: %v", indices)
+		}
+	}
+
+	{
+		val, indices := util.FindFlagVal(
+			"--logLevel", "rexray", "--logLevel=")
+		if val != "" {
+			t.Fatalf("val != '': %s", val)
+		}
+		if len(indices) != 1 && indices[0] != 1 {
+			t.Fatalf("invalid indices: %v", indices)
+		}
+	}
 }

